pkg/identityserver/entity_access.go

```diff
@@ -39,6 +39,9 @@ var (
 	errInvalidAuthorization     = errors.DefineUnauthenticated("invalid_authorization", "invalid authorization")
 	errTokenNotFound            = errors.DefineUnauthenticated("token_not_found", "access token not found")
 	errTokenExpired             = errors.DefineUnauthenticated("token_expired", "access token expired")
+	errRejectedUser             = errors.DefineUnauthenticated("user_rejected", "user account approval is rejected")
+	errRequestedUser            = errors.DefineUnauthenticated("user_requested", "user account approval is pending")
+	errSuspendedUser            = errors.DefineUnauthenticated("user_suspended", "user account is suspended")
```

Comment:
Please keep consistent with the OAuth errors below:

- `errUserXxx` instead of `errXxxUser`
- remove "approval" on the rejected error
- "is" -> "was" on rejected and suspended errors